app/javascript/css/main.css.scss

```diff
@@ -34,8 +34,8 @@ a {
   flex-wrap: wrap;
   align-items: center;
   border-bottom: 2px solid $gray;
-  padding-left: 11.88%;
-  padding-right: 11.88%;
+  padding-left: 13.5%;
+  padding-right: 13.5%;
```

Comment:
can you change padding-left and padding-right to 9%

Reply:
remove this changes